pkg/console/starter/starter.go

```diff
@@ -174,6 +176,19 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		ctx.EventRecorder,
 	)
 
+	staleConditionsController := staleconditions.NewRemoveStaleConditions(
```

Comment:
For reference, we see this elsewhere:
https://github.com/openshift/cluster-authentication-operator/pull/159/commits/17a0064a2c10bbf1e2b5926f21192ed5153cb5e2#diff-c9f95324de2319f30fff1ad2fd4e7b00R160